backend/lib/services/members.js

```diff
@@ -236,12 +347,15 @@ exports.remove = async function ({ user, namespace, name }) {
 
   // unassign user from project
   const project = await unsetProjectMember(client, { namespace, name })
-  const [, serviceaccountNamespace, serviceaccountName] = /^system:serviceaccount:([^:]+):([^:]+)$/.exec(name) || []
-  if (serviceaccountNamespace === namespace) {
-    await deleteServiceaccount(client, {
-      namespace,
-      name: serviceaccountName
-    })
+
+  if (hasServiceAccountPrefix(name)) {
+    const { serviceAccountNamespace, serviceAccountName } = prefixedServiceAccountToComponents(name)
+    if (serviceAccountNamespace === namespace) {
+      await deleteServiceaccount(client, {
```

Comment:
we have some inconsistencies in our code base
serviceaccount vs serviceAccount

Reply:
should be camel case, will check